app/utils/auth.py

```diff
@@ -14,11 +14,13 @@ def access_secret_file(secret_id, version_id="latest"):
 
 # Function to ensure incoming request is from controller with key
 def key_check(api_key: str = Header(None)):
-  
+  print("api key",api_key)
```

Comment:
Do not print the environment variables anywhere. It is unsafe and is not good practice.